approval_polls/templates/approval_polls/edit.html

```diff
@@ -0,0 +1,124 @@
+  <script src='{% static 'vendor/jquery-datetimepicker/js/jquery.datetimepicker.full.min.js' %}'></script>
+  <script src='{% static 'vendor/bootstrap-tokenfield/js/bootstrap-tokenfield.min.js' %}'></script>
+  <script src='{% static 'approval_polls/create.js' %}'></script>
+  <script src='{% static 'approval_polls/edit.js' %}'></script>
+{% endblock %}
+
+{% block content %}
+
+<div class='row-fluid full-height'>
+  <div class='col-md-12'>
+    <form action="{% url 'approval_polls:edit' poll_id=poll.id %}" method='post'>
+      <legend> Edit: {{poll.question}}</legend>
+      {% csrf_token %}
+<!--
+      <div class='form-group {% if question_error %} has-error {% endif %}'>
+        {% if question_error %}
+          <label class='control-label' for='question'>
+              {{ question_error }}
+          </label>
+        {% endif %}
+        <input type='text' class='form-control' maxlength='200' value='{{ poll.question }}'
+               name='question' size='28' placeholder='Poll Question'>
+      </div>
+-->
+<!--
+      {% for choice in choices %}
+      <div class='form-group'>
+        <div class='input-group' id='div-choice{{ choice.pk }}'>
+          <input class='form-control' type='text' maxlength='200' name='choice{{ choice.pk }}' placeholder='Choice {{choice.pk}}' value='{{choice.choice_text}}'>
+          <span class='input-group-addon'>
+            <a href='#' class='add-link' id='link-choice{{ choice.pk }}' title='Add link' data-toggle='tooltip' data-placement='bottom'>
+              <span class='glyphicon glyphicon-link'></span>
+            </a>
+          </span>
+          <span class='input-group-addon'>
+            <a href='#' class='remove-choice' id='remove-choice{{ choice.pk }}' title='Remove Choice' >
+              <span class='glyphicon glyphicon-remove'></span>
+            </a>
+          </span>
+          <input type='hidden' id='linkurl-choice{{ choice.pk }}' name='linkurl-choice{{ choice.pk }}' value=''>
+        </div>
+      </div>
+      {% endfor %}
+-->
+      <p>
+        <button type='button' class='btn btn-primary btn-sm' id='add-choice'
+              title='Add choice' data-toggle='tooltip' data-placement='bottom'>
+          <i class='glyphicon glyphicon-plus'></i>
+        </button>
+      </p>
+      <!-- Add the collapsible poll settings panel here -->
+      <div class='panel-group' id='accordion' role='tablist' aria-multiselectable='true'>
+        <div class='panel panel-default'>
+          <div class='panel-heading' role='tab' id='headingOne'>
+            <h4 class='panel-title'>
+              <a role='button' data-toggle='collapse' data-parent='#accordion' href='#collapsePollSettings' aria-expanded='true' aria-controls='collapsePollSettings'>
+               Poll Settings
+              </a>
+            </h4>
+          </div>
+<!--
+          <div id='collapsePollSettings' class='panel-collapse' role='tabpanel' aria-labelledby='headingOne'>
+            <div class='panel-body'>
```

Comment:
Currently the controls do not seem to have a margin offset from the panel border. You could keep this div for the controls to get some offset from the border.

>           <div role='tabpanel' aria-labelledby='headingOne'>
>             <div class='panel-body'>

 (PS: I am viewing them in chrome)